src/style.css

```diff
@@ -1 +1,43 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
The selector '.stars--0 .stars__star:nth-child(-n + 0)' will not apply any styles because '-n + 0' does not select any elements. If the intention is to have no active stars for '.stars--0', this is correct. Otherwise, you might want to adjust this to select the appropriate number of stars.